freqtrade/exchange/exchange.py

```diff
@@ -1525,6 +1581,61 @@ def get_historic_trades(self, pair: str,
             self._async_get_trade_history(pair=pair, since=since,
                                           until=until, from_id=from_id))
 
+    @retrier
+    def fill_leverage_brackets(self):
+        """
+            #TODO-lev: Should maybe be renamed, leverage_brackets might not be accurate for kraken
+            Assigns property _leverage_brackets to a dictionary of information about the leverage
+            allowed on each pair
+        """
+        raise OperationalException(
+            f"{self.name.capitalize()}.fill_leverage_brackets has not been implemented.")
+
+    def get_max_leverage(self, pair: Optional[str], nominal_value: Optional[float]) -> float:
+        """
+            Returns the maximum leverage that a pair can be traded at
+            :param pair: The base/quote currency pair being traded
+            :nominal_value: The total value of the trade in quote currency (collateral + debt)
+        """
+        raise OperationalException(
+            f"{self.name.capitalize()}.get_max_leverage has not been implemented.")
```

Comment:
It's unclear to me at the moment where you plan on calling `get_max_leverage()` - but woulnd't it make sense to return "1" (or 0 - whatever means "No leverage") instead of raising here? 
that might make using this method a little easier ...